tests/src/CLRTest.Execute.Batch.targets

```diff
@@ -72,20 +72,23 @@ WARNING:   When setting properties based on their current state (for example:
 $(BatchCLRTestEnvironmentCompatibilityCheck)
 IF NOT "%COMPlus_GCStress%"=="" (
   ECHO SKIPPING EXECUTION BECAUSE COMPlus_GCStress IS SET
+  popd
```

Comment:
(and yes, that is the corresponding pushd. I have found myself needing to repeatedly popd after running tests)